iOS/Flows/Conversation/Cells/ConversationMessageCell.swift

```diff
@@ -299,7 +212,7 @@ extension ConversationMessageCell: UICollectionViewDataSource, UICollectionViewD
                         layout collectionViewLayout: UICollectionViewLayout,
                         referenceSizeForHeaderInSection section: Int) -> CGSize {
         // Only show a header for the first section
-        if section == 0 && !self.otherMessages.isEmpty {
+        if section == 0 && self.dataSource.snapshot().numberOfItems(inSection: .otherMessages) != 0 {
```

Comment:
I'm going to make some convenience functions for getting the number of items later.